src/api/endpoints/analytics.py

```diff
@@ -1,12 +1,14 @@
 from fastapi import APIRouter, Depends
 
 from src.api.schemas import Statistic
-from src.api.services.analitics import get_number_users_statistic
+from src.api.services.analitics import AnalyticsService
 
 analytic_router = APIRouter()
 
 
-@analytic_router.get("/", description="Возращает статистику сервиса.")
-async def get_statistics() -> Statistic:
-    statistics = Statistic.number_users = get_number_users_statistic()
+@analytic_router.get("/", response_model=Statistic, description="Возращает статистику сервиса.")
+async def get_statistics(analytic_service: AnalyticsService = Depends()) -> Statistic:
+    users_number = await analytic_service.get_users_number()
+    statistics = Statistic
+    statistics.number_users = users_number
```

Comment:
Это очень плохая практика, так будет лучше
```suggestion
    statistics  = Statistic(number_users=await analytic_service.get_users_number())
```
Также рекомендую использовать меньше переменных - в идеале только если они используется несколько раз в коде. Так будет гораздо легче читать код.